config/dev.webpack.config.js

```diff
@@ -1,10 +1,19 @@
+// eslint-disable-next-line max-len
+// Reference link: https://github.com/RedHatInsights/frontend-components/tree/master/packages/config#redhat-cloud-services-frontend-components---webpack-config
+
 const { resolve } = require('path');
 const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
     rootFolder: resolve(__dirname, '../'),
     debug: true,
     https: true,
-    ...(process.env.BETA && { deployment: 'beta/apps' })
+    appUrl: process.env.BETA ? '/beta/insights/ros' : '/insights/ros',
+    env: `${process.env.ENVIRONMENT || 'stage'}-${process.env.BETA ? 'beta' : 'stable'}`,
+    deployment: process.env.BETA ? 'beta/apps' : 'apps',
+    useProxy: true,
+    proxyVerbose: true,
+    useChromeTemplate: true,
```

Comment:
@karelhala based on the documentation [here](https://github.com/RedHatInsights/frontend-components/blob/master/packages/config/README.md#usechrometemplate) what I understand that after enabling this we won't need local chrome anymore? and in case we want to build with local chrome we would provide the path to it (below is config for that) did I get it right?

Reply:
The `useChromeTemplate` flag means that the HTML template the browser uses is from chrome, instead from the application itself.

If you want to use local chrome, you build chrome locally and configure the proxy with (absolute) path to the `/chrome/build` directory.

Reply:
Thanks @Hyperkid123 for sharing the information, I am just curios about in which cases we would need to use local chrome? is it when we want to test the chrome changes which are not deployed yet?

Reply:
>  is it when we want to test the chrome changes which are not deployed yet?
Exactly. You also may want to use local chrome when running the react profiler. As it requires a development version of React which you only have when running with local chrome. Apps are not using locally installed React and ReactDOM. They use the instance served by chrome.